### PR TITLE
Feature globalization

### PR DESCRIPTION
**Pull request by baikauskaite**
Implemented globalization

* Added localization services in Startup.cs
* Implemented resource files for DonateDialog, RequestDialog, _Layout views.
* Implemented localization for DonateDialog, RequestDialog, _Layout views.
* Added TagHelper for localization in _ViewImports.cshtml
* Fixed unintended comment in _Layout.cshtml

### CODE REVIEW

**Review by PauliusLu — APPROVED** [review]
Lovely

File: Karma/Views/Home/DonateDialog.cshtml [hunk 1]
```diff
@@ -48,15 +49,15 @@
...
+                    @localizer["createPostPhrase"]
                 </div>
             </div>
             @*TODO: redirect to page (dialog?) for donate post creation*@
             <div class="row">
                 <button type="button" class="btn btn-light col">
                     <div class="row align-items-center">
                         <div class="col-12 text-center font-weight-bold">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Posts" asp-action="Donate">Create a donation post</a>
+                            @localizer["createPost"]
```
Comment (ArmintasP) [inline Karma/Views/Home/DonateDialog.cshtml:60]: The link would stop working and when button is pressed it won't redirect to `/Posts/Donate`.

File: Karma/Views/Home/RequestDialog.cshtml [hunk 2]
```diff
@@ -36,15 +37,15 @@
...
+                    @localizer["createRequestPhrase"]
                 </div>
             </div>
             @*TODO: redirect to page (dialog?) to create a request*@
             <div class="row">
                 <button type="button" class="btn btn-light col">
                     <div class="row">
                         <div class="col-12 text-center font-weight-bold">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Posts" asp-action="CreateRequest">Create a request</a>
+                            @localizer["createRequest"]
```
Comment (ArmintasP) [inline Karma/Views/Home/RequestDialog.cshtml:48]: The link would stop working and when button is pressed it won't redirect to `/Posts/CreateRequest`.

**Review by ArmintasP — COMMENTED** [review]
Button/link bug should be fixed.

**Pull request merged**